crates/pallet-domains/src/lib.rs

```diff
@@ -1287,7 +1287,7 @@ mod pallet {
             match call {
                 Call::submit_bundle { opaque_bundle } => {
                     if let Err(e) = Self::validate_bundle(opaque_bundle) {
-                        log::debug!(
+                        log::info!(
```

Comment:
Seems like unrelated changes

Reply:
I would like the errors to be printed out instead of gobbled up. If there are other places where mark them debug, we should mark then info! instead debug! so that we dont need to restart the node with debug.

This is a related change I noticed during the tests

Reply:
Yeah, printing the error out does help a lot in debugging, we should also change the log level for `validate_fraud_proof`, and perhaps `warn!` or `error!` will be more appropriate, it may be out of the scope of this PR we can do that in a separate PR.